netlink-packet-route/src/rtnl/link/nlas/af_spec_bridge.rs

```diff
@@ -0,0 +1,67 @@
+// SPDX-License-Identifier: MIT
+use anyhow::Context;
+
+use crate::{
+    constants::*,
+    nlas::{self, DefaultNla, NlaBuffer, NlasIterator},
+    parsers::parse_u16,
+    traits::{Emitable, Parseable},
+    DecodeError,
+};
+
+use byteorder::{BigEndian, ByteOrder, NativeEndian};
+
+#[derive(Clone, Eq, PartialEq, Debug)]
+pub enum AfSpecBridge {
+    Flags(u16),
+    VlanInfo(Vec<u8>),
+    Other(DefaultNla),
+}
+
+impl nlas::Nla for AfSpecBridge {
+    #[rustfmt::skip]
```

Comment:
I don't think we need to skip rustfmt here. The formatted output should be the same.